src/main/java/org/assertj/guava/api/RangeSetAssert.java

```diff
@@ -0,0 +1,982 @@
+ *       or other liability obligations and/or rights consistent with this
+ *       License. However, in accepting such obligations, You may act only
+ *       on Your own behalf and on Your sole responsibility, not on behalf
+ *       of any other Contributor, and only if You agree to indemnify,
+ *       defend, and hold each Contributor harmless for any liability
+ *       incurred by, or claims asserted against, such Contributor by reason
+ *       of your accepting any such warranty or additional liability.
+ *
+ *    END OF TERMS AND CONDITIONS
+ *
+ *    APPENDIX: How to apply the Apache License to your work.
+ *
+ *       To apply the Apache License to your work, attach the following
+ *       boilerplate notice, with the fields enclosed by brackets "[]"
+ *       replaced with your own identifying information. (Don't include
+ *       the brackets!)  The text should be enclosed in the appropriate
+ *       comment syntax for the file format. We also recommend that a
+ *       file or class name and description of purpose be included on the
+ *       same "printed page" as the copyright notice for easier
+ *       identification within third-party archives.
+ *
+ *    Copyright [yyyy] [name of copyright owner]
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package org.assertj.guava.api;
+
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.util.VisibleForTesting;
+import org.assertj.guava.internal.RangeSets;
+
+import com.google.common.collect.Range;
+import com.google.common.collect.RangeSet;
+
+/**
+ * Assertion for guava {@link com.google.common.collect.RangeSet}.
+ * <p>
+ * To create an instance of this class, invoke <code>{@link
+ * org.assertj.guava.api.Assertions#assertThat(com.google.common.collect.RangeSet)}</code>
+ * <p>
+ *
+ * @param <T> the type of the tested RangeSet elements
+ * @author Ilya Koshaleu
+ */
+public class RangeSetAssert<T extends Comparable<T>> extends AbstractAssert<RangeSetAssert<T>, RangeSet<T>> {
+
+  @VisibleForTesting
+  RangeSets rangeSets = RangeSets.instance();
+
+  protected RangeSetAssert(RangeSet<T> actual) {
+    super(actual, RangeSetAssert.class);
+  }
+
+  /**
+   * Verifies that the given {@code RangeSet} has specific {@code size} of disconnected {@code Range} elements.
+   * <p>
+   * Example:
+   *
+   * <pre><code class='java'> 
+   * RangeSet&lt;Integer&gt; rangeSet = TreeRangeSet.create();
```

Comment:
code examples must start one space after: `<pre><code class='java'>` like this:
```
<pre><code class='java'> RangeSet&lt;Integer&gt; rangeSet = TreeRangeSet.create(); 
```

Reply:
applies everywhere

Reply:
fixed